### ISSUE TITLE
Custom website.

### ISSUE THREAD

**Issue by teymour-aldridge** [yewstack/docs#71]
I think it would be cool for Yew to have a custom website instead of using Gitbook. The advantage to this is that it would help improve the image (which is already good) of the project to make it appear more established.

I think that this would best be done by designing a custom mdbook template. 

I'd be happy to work on this.

**Comment by jstarry** [comment]
Agreed, I think it's a great usecase for static site generation

**Comment by teymour-aldridge** [comment]
Yes (we could even integrate Yew into it if the static site generation feature gets of the ground).
Hosting could be done on Github pages?

**Comment by jstarry** [comment]
> Yes (we could even integrate Yew into it if the static site generation feature gets of the ground).

Yes, exactly what I meant :)

**Comment by teymour-aldridge** [comment]
Oh right :)

**Comment by teymour-aldridge** [comment]
I've got a draft template in PR #72.
